src/main/resources/db/migration/V2__Add_employee_table.sql

```diff
@@ -0,0 +1,11 @@
+create type gender as enum ('M', 'F');
+
+create table employees
+(
+    emp_no     Integer     not null primary key,
+    birth_date date        not null,
+    first_name varchar(14) not null,
+    last_name  varchar(16) not null,
+    type       gender      not null,
```

Comment:
```suggestion
    gender       gender      not null,
```

The column name should be `gender` as in the image I sent you